docs/admin.rst

```diff
@@ -418,33 +418,42 @@ Updating System Configuration
 
 .. _update-system-configuration:
 
-If you want to update the system configuration, you should use the
-``securedrop-admin`` tool on the *Admin Workstation*. From
-``~/Persistent/securedrop``, run:
+There are two primary reasons why you may want to update the system configuration:
 
-.. code:: sh
+- to change SecureDrop server configuration options. **Example:** You want to change
+  the time of day at which the servers are automatically rebooted (default: 4:00 AM).
+- to fix a problem with the configuration state of your *Application* and/or
+  *Monitor Server*. **Example:** An administrator has locally modified your iptables
```

Comment:
This example is a bit contrived, and might give admins the impression that modifications to iptables are something that they are expected to do. Something like `OSSEC SMTP settings are incorrect and need to be updated.` might make more sense

Reply:
I'd like to distinguish between
- a configuration change that you or someone else made explicitly via `securedrop-admin sdconfig` vs.
- the system state that is enforced by the playbook.

It's IMO important to understand that running `securedrop-admin install` does more than just applying your site-specific configuration settings. And the reason I picked `iptables` as an example here is to make it easier for folks to recognize this section as relevant if they jump to it from the other chapter.

We could say something like "an administrator unfamiliar with the system" or other language to make it clear that such edits aren't desirable.

Reply:
done in aa2d96968dc8da9d4556f9836273f2269ad95180

Reply:
Maybe instead of "to fix a problem..." the bulletpoint could just be "to restore the servers' configuration to a valid state. **Example:** to revert networking changes made for troubleshooting purposes."

Reply:
How about:
> to restore a valid configuration state on your servers. Example: Another admin has directly modified the iptables rules during troubleshooting, and you want to reinstate the correct rules.

Reply:
sgtm

Reply:
done in 359a6194e0cbbaca6b3a03b0c223adb8f4a4c5ad